orderbook/src/api/create_order.rs

```diff
@@ -81,8 +81,8 @@ pub fn create_order_response(result: Result<AddOrderResult>) -> impl Reply {
         Ok(AddOrderResult::TransferEthToContract) => (
             super::error(
                 "TransferEthToContract",
-                "Setting receiver to a smart contract wallet when buying Ether \
-                 is currently not supported",
+                "Sending Ether to a smart contract wallets is currently not \
+                 supported",
```

Comment:
Is it ever going to be supported?

Reply:
I think so, it's technically possible by including the SC wallet address in the access list.

Reply:
It is possible. I had a tenderly simulation with access list set where I successfully transferred Eth to Gnosis Safe somewhere. We just need to add support for EIP-2930 access lists and set the appropriately when sending Eth to SCs.